### PR TITLE
Adding Acceptance tests for Budget pages

### PR DESCRIPTION
**Pull request by sebworks**
Adding Acceptance tests for the `Budget` pages

## Changes
 - Added acceptance tests for the `Budget` pages. 

## Testing
-  Run `gulp test:acceptance`

## Review
@anselmbradford 
@jimmynotjim 
@KimberlyMunoz

### CODE REVIEW

**Comment by KimberlyMunoz** [comment]
LGTM and all tests pass :+1:

File: test/browser_tests/page_objects/page_budget.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    browser.get( '/budget/' );
+  };
+
+  this.pageTitle = function() { return browser.getTitle(); };
+
+  this.sideNav = element( by.css( '.nav-secondary' ) );
+
+  this.mainTitle = element( by.css( '.media_body h1' ) );
+
+  this.missionSummary = element( by.css( '[data-qa-hook="mission-summary"]' ) );
```
Comment (jimmynotjim) [inline test/browser_tests/page_objects/page_budget.js]: Use `_getQAelement` here and throughout

**Comment by jimmynotjim** [comment]
Should [this comment](https://github.com/cfpb/cfgov-refresh/pull/1005#discussion_r40157279) apply to the mappings here too?

**Comment by anselmbradford** [comment]
:+1:

**Comment by jimmynotjim** [comment]
:+1:

**Pull request merged**